config/default.js

```diff
@@ -272,4 +273,6 @@ module.exports = {
 
   // See the Discopane config for how this is currently used.
   taarParamsToUse: [],
+
+  betaVersions: true,
```

Comment:
Since this config key is a bit vague, I'd like to see some comments explaining how this controls add-on beta version features.